content/md/what-is/what-about-reference-entities.md

```diff
@@ -1,24 +1,25 @@
 ---
 id: what-about-reference-entities
 themes: first-steps, manage-your-reference-entities
-title: What about **reference entities**?
+title: What is a **reference entity**?
 popular: true
 ee-only: true
 related: manage-reference-entities, enrich-your-reference-entity-records
 ---
 
-# What's the use case?
+# Introduction to reference entities
 
-Let's take an example of a product page.  
+In order to present you quickly this attribute type, let's take an example of a product page.  
```

Comment:
It's not only an attribute type, it's a new entity / object like the product object.

Reply:
```suggestion
In order to present you quickly this new object type, let's take an example of a product page.  
```

Reply:
Bon au final "object" ça fait bizarre aussi, j'ai mis "reference entities"